monkey/monkey_island/cc/services/telemetry/processing/system_info.py

```diff
@@ -20,6 +21,7 @@ def process_system_info_telemetry(telemetry_json):
         process_aws_data,
         update_db_with_new_hostname,
         test_antivirus_existence,
+        process_environment_telemetry
```

Comment:
Why does this get a package and the rest don't? Future janitoring?

Reply:
Yep